tensorflow-framework/src/main/java/org/tensorflow/framework/op/nn/SigmoidCrossEntropyWithLogits.java

```diff
@@ -1,29 +1,32 @@
-package org.tensorflow.op.nn;
+package org.tensorflow.framework.op.nn;
 
 import org.tensorflow.Operand;
 import org.tensorflow.ndarray.Shape;
 import org.tensorflow.op.Scope;
-import org.tensorflow.op.annotation.Endpoint;
-import org.tensorflow.op.annotation.Operator;
 import org.tensorflow.op.core.Select;
 import org.tensorflow.op.core.ZerosLike;
 import org.tensorflow.op.dtypes.Cast;
-import org.tensorflow.op.math.*;
+import org.tensorflow.op.math.Add;
+import org.tensorflow.op.math.Exp;
+import org.tensorflow.op.math.GreaterEqual;
+import org.tensorflow.op.math.Log1p;
+import org.tensorflow.op.math.Mul;
+import org.tensorflow.op.math.Neg;
+import org.tensorflow.op.math.Sub;
 import org.tensorflow.types.TBool;
 import org.tensorflow.types.family.TNumber;
 
-@Operator(group = "nn")
+// @Operator(group = "nn")
```

Comment:
You probably want to remove those commented lines...

Reply:
I have removed all @Operator annotations, unless @rnett needs them

Reply:
Probably not, I'll look at how to handle framework ops in Kotlin but that wasn't a particularly good way in the first place.  Worse case I can re-add them.